src/components/GroupedTaskStatusBadge/GroupedTaskStatusBadge.stories.tsx

```diff
@@ -37,3 +47,18 @@ const Container = styled.div`
   flex-direction: row;
   justify-content: space-around;
 `;
+
+const statusCounts = {
+  started: 30,
+  failed: 15,
+  unstarted: 5,
+  unscheduled: 6,
+  "will-run": 11,
+  dispatched: 99,
+  pending: 987,
+  "test-timed-out": 2,
+  "task-timed-out": 53,
+  "system-failed": 22,
```

Comment:
The system failures badge on the storybook is looking a bit funky:
<img width="78" alt="image" src="https://user-images.githubusercontent.com/9298431/136278902-5bfcdaf0-e3d8-49c2-9cb4-996da065ca52.png">